File: Majo/Act-Instructor_Henry/lenguajes/dart/orientado_a_objeto/clase01.dart
class Cat{
  String? raza;
  int? edad;
  String? sexo;
}

class Maquillaje{
  String? marca;
  String? producto;
  int? precio;
}

class Serie {
  String? titulo;
  String? creador;
  int? temporadas;
}


void main(){
  Cat x = new Cat();

  x.raza = "Siames";
  x.edad = 3;
  x.sexo = "Macho";

  Maquillaje y = new Maquillaje();

  y.marca = "Rare Beauty";
  y.producto = "Blush";
  y.precio = 170000;

  Serie z = new Serie();

  z.titulo = "Gossip Girl";
  z.creador = "Josh Schwartz y Stephanie Savage";
  z.temporadas = 7;  
}

File: Majo/Act-Instructor_Henry/lenguajes/ruby/orientado_a_objeto/clase01.rb
class Cat
    attr_accessor :raza, :edad, :sexo
end
  
class Maquillaje
    attr_accessor :marca, :producto, :precio
end
  
class Serie
    attr_accessor :titulo, :creador, :temporadas
end
  

x = Cat.new
x.raza = "Siames"
x.edad = 3
x.sexo = "Macho"
  
y = Maquillaje.new
y.marca = "Rare Beauty"
y.producto = "Blush"
y.precio = 170000
 
z = Serie.new
z.titulo = "Gossip Girl"
z.creador = "Josh Schwartz y Stephanie Savage"
z.temporadas = 7